distros/spring/src/test/java/com/connexta/transformation/service/TransformControllerTest.java

```diff
@@ -47,13 +50,13 @@
 
   @Autowired private MockMvc mockMvc;
 
-  @MockBean TransformationManager tManager;
-
-  @MockBean(answer = Answers.RETURNS_MOCKS)
-  private RestTemplateBuilder restTemplateBuilder;
+  @MockBean InMemoryTransformationManager manager;
```

Comment:
✏️ Mock the Interface instead of the implementation.

Reply:
fixed